books/free-programming-books-id.md

```diff
@@ -158,10 +158,13 @@
 ### Python
 
 * [Belajar Python](http://www.belajarpython.com)
+* [Belajar python bahasa Indonesia](https://sekolahkoding.com/track/belajar-python-bahasa-indonesia) - Sekolah Koding
 * [Dasar Pemrograman Python](https://www.pythonindo.com/tutorial-python-dasar/)
 * [Dasar Pengenalan Pemrograman Python](https://play.google.com/store/books/details/Rolly_Maulana_Awangga_Dasar_dasar_Python?id=YpzDDwAAQBAJ) - Rolly Maulana AwanggaRayhan *(Membutuhkan akun Google Play Books atau email yang valid)*
 * [Database PostgreSQL, Pemrograman Python, dan SMS Gateway](http://rab.co.id/files/python/bukupython2.pdf.gz) (PDF)
 * [Kursus Singkat Machine Learning dengan TensorFlow API](https://developers.google.com/machine-learning/crash-course?hl=id)
+* [Python](https://codesaya.com/python/) - CodeSaya
```

Comment:
I would characterize this as a 1. interactive tutorial or 2. a course or 3. not really a book.

Reply:
got it, let me put in to course instead. thanks for the correction!